pandas/tests/series/test_timeseries.py

```diff
@@ -415,6 +415,10 @@ def test_pct_change(self):
             rs, (filled / filled.shift(freq="5D") - 1).reindex_like(filled)
         )
 
+        rs = self.ts.pct_change(freq="1W")
```

Comment:
Can you make this its own test? Also how does this vary from the `5D` output directly above this in the module?

Reply:
Sure, I can.
 
![image](https://user-images.githubusercontent.com/19762154/65934000-aea2d600-e44e-11e9-890e-ae6c075f6253.png)

even though the actual periods are the same, there is a difference between them like above.
So I added the line 418 `rs = self.ts.pct_change(freq="1W")` to test also [anchored freq](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.tseries.offsets.DateOffset.isAnchored.html#pandas.tseries.offsets.DateOffset.isAnchored)(such as1W, 1M, BM).